src/WarriorRogueMage/CharacterCreation/SetAttributesState.cs

```diff
@@ -189,15 +189,18 @@ private SetAttributeCommand FindTargetCommand(string s)
             {
                 return SetAttributeCommand.Warrior;
             }
-            else if (s.StartsWith("r"))
+
+            if (s.StartsWith("r"))
             {
                 return SetAttributeCommand.Rogue;
             }
-            else if (s.StartsWith("m"))
+
+            if (s.StartsWith("m"))
```

Comment:
The "else if" pattern should compile to an optimized result, but more importantly ensures expansion of options doesn't end up with a situation where we accidentally introduce a bug where, say, "no" and "none" are two different commands that both execute when typing "no"

Reply:
In this context it would never happen because of the return.

Reply:
Ah, whoops, of course.  :)